### PR TITLE
Update Brave Talk study to be in Release

### PR DESCRIPTION
**Pull request by bsclifton**
Per today's triage (Sep 7th), here's the uplift for Talk to RELEASE channel

cc: @jsecretan for the go-ahead. Was uplifted / tested against staging here:
https://github.com/brave/brave-variations/pull/93

### CODE REVIEW

**Review by rebron — APPROVED** [review]
Looks good. ++

File: seed/seed.json [hunk 1]
```diff
@@ -913,7 +913,7 @@
                 }
             ],
             "filter": {
-                "channel": ["BETA", "NIGHTLY"],
```
Comment (iefremov) [inline seed/seed.json:916]: typically we should specify min_version, to not enable it for old clients @bsclifton @rebron

Comment (iefremov) [reply seed/seed.json:916]: also @kjozwiak

Comment (kjozwiak) [reply seed/seed.json:916]: @iefremov can we get this fixed or is it too late at this point? What would happen if a client that ended up pulling the above study doesn't meet the new requirements. Would it simply be removed? Assuming it wouldn't cause any issues for users that meet the requirements and have the study?

I'll make sure we have a `min_version` next time 👍

Comment (bsclifton) [reply seed/seed.json:916]: Thanks for the heads up - will do that going forward. No action needed for this, AFAIK. But they would be shown as being in an experiment which isn't applicable

**Pull request merged**